bundle/store.go

```diff
@@ -222,6 +226,27 @@ func readRevisionFromStore(ctx context.Context, store storage.Store, txn storage
 	return str, nil
 }
 
+// ReadBundleRevisionFromStore returns the metadata in the specified bundle.
+// If the bundle is not activated, this function will return
+// storage NotFound error.
+func ReadBundleMetadataFromStore(ctx context.Context, store storage.Store, txn storage.Transaction, name string) (map[string]interface{}, error) {
```

Comment:
I don't see any test calling this. Is it called indirectly?

Reply:
No, thanks for the hint, I've updated tests.